wavelength-plugins/misc.js

```diff
@@ -422,7 +422,21 @@ exports.commands = {
 			if (!Rooms(roomid)) return this.errorReply(`${roomid} is not a room.`);
 			if (Db.disabledScrolls.has(roomid)) return this.errorReply(`${Rooms(roomid).title} has already roomintro scroll disabled.`);
 			msg += '/html <center>' + WL.nameColor(user.name, true) + ' has redeemed roomintro scroll disabler token.<br/>';
-			msg += '<button class="button" name="send" value="/disableintroscroll ' + target[1] + '">Disable Intro Scrool for <b>' + Rooms(roomid).title + '</b></button></center>';
+			msg += '<button class="button" name="send" value="/disableintroscroll ' + target[1] + '">Disable Intro Scroll for <b>' + Rooms(roomid).title + '</b></button></center>';
+			delete user.tokens[target[0]];
+			return WL.messageSeniorStaff(msg);
+		case 'background':
+			if (!target[1]) return this.errorReply('/usetoken background, [img]');
+			target[1] = target[1].trim();
+			msg += '/html <center>' + WL.nameColor(user.name, true) + ' has redeem a background token.<br/><img src="' + target[1] + '"/><br/>';
+			msg += '<button class="button" name="send" value="/background set ' + user.userid + ', ' + target[1] + '">Set the background</button></center>';
+			delete user.tokens[target[0]];
+			return WL.messageSeniorStaff(msg);
+		case 'music':
+			if (!target[2]) return this.errorReply('/usetoken music, [link], [name]');
+			target[1] = target[1].trim();
+			msg += '/html <center>' + WL.nameColor(user.name, true) + ' has redeem a music token.<br/><audio src="' + target[2] + '" alt="' + target[1] + '"></audio><br/>';
```

Comment:
Check a music link to make sure its valid

Reply:
Don't know how to index to check that tbh